docs/migrating-to-v4-guide.md

```diff
@@ -269,3 +284,70 @@ test('testing-library/custom-renders setting example', () => {
   const invalidUsage = setupB(<Component />);
 });
 ```
+
+You can also set this setting to `"off"` to entirely opt-out Aggressive Renders Reporting mechanism, so only methods named `render` are reported as Testing Library render util.
+
+```json
+// .eslintrc
+{
+  "settings": {
+    "testing-library/custom-renders": "off"
+  }
+}
+```
+
+### `testing-library/custom-queries`
+
+Relates to [Aggressive Queries Reporting mechanism](#queries). This setting accepts an array of strings or `"off"`.
+
+If you pass an array of strings to this option, it will represent a list of query names/variants that are the only valid Testing Library custom queries.
+
+Each string passed to this list of custom queries can be:
+
+- **pattern query (recommended)**: a custom query variant (suffix starting with "By") to be reported, so all query combinations around it are reported. For instance: `"ByIcon"` would report all `getByIcon()`, `getAllByIcon()`, `queryByIcon()` and `findByIcon()`.
+- **strict query**: a specific custom query name to be reported, so only that very exact query would be reported but not any related variant. For instance: `"getByIcon"` would make the plugin to report `getByIcon()` but not `getAllByIcon()`, `queryByIcon()` or `findByIcon()`.
+
+```json
+// .eslintrc
+{
+  "settings": {
+    "testing-library/custom-queries": ["ByIcon", "getByComplexText"]
+  }
+}
+```
+
+Configuring this setting like that, you'll restrict the errors reported by the plugin related to the queries somehow to only those custom queries matching name or pattern from that list, or [built-in queries](https://testing-library.com/docs/queries/about). The previous setting example would cause:
```

Comment:
```suggestion
Configuring this setting like that, you'll restrict the errors reported by the plugin related to the queries to only those custom queries matching name or pattern from that list, or [built-in queries](https://testing-library.com/docs/queries/about). The previous setting example would cause:
```

Reply:
Thanks for all the grammar suggestions, better than Grammarly 😅